medusa/tv/series.py

```diff
@@ -2021,7 +2022,15 @@ def to_json(self, detailed=True):
         if self.is_anime:
             data['config']['release']['blacklist'] = bw_list.blacklist
             data['config']['release']['whitelist'] = bw_list.whitelist
-            data['config']['release']['allgroups'] = get_release_groups_for_anime(self.name)
+            try:
+                data['config']['release']['allgroups'] = get_release_groups_for_anime(self.name)
+            except AnidbAdbaConnectionException:
+                data['config']['release']['allgroups'] = []
+                log.debug(
+                    'An anidb adba exception occurred when attempting to get the release groups for the show {show}',
+                    {'show': self.name}
```

Comment:
Well it's not really what I meant,
I suggested you add the actual error to the log...